src/qt_gui/kbm_help_dialog.cpp

```diff
@@ -267,3 +271,35 @@ You can find these here, with detailed comments, examples, and suggestions for m
 'mouse_gyro_roll_mode':
     Controls whether moving the mouse sideways causes a panning or a rolling motion while mouse-to-gyro emulation is active.)";
 }
+
+QString HelpDialog::faq() {
```

Comment:
```suggestion
constexpr QString HelpDialog::faq() {
```
Same for the other functions too

Reply:
I can't add constexpr because QString isn't a literal data type.

Reply:
No, you couldn't add constexpr because you forgot to update the header